README.md

```
@@ -1,568 +1,590 @@
+        .withSubChart(macd) // MACD indicator in subchart
+        .withSubChart(rsi) // RSI indicator in subchart
+        .withSubChart(new NetProfitLossCriterion(), record) // Net profit/loss performance metric
+        .toChart();
+```
+<!-- END_SNIPPET: advanced-strategy -->
+
+![Advanced Multi-Indicator Strategy](ta4j-examples/docs/img/advanced-strategy-readme.jpg)
+
+This comprehensive chart demonstrates combining multiple indicators (MACD, RSI) in separate subcharts with performance metrics, giving you a complete view of strategy behavior.
+
+See the [chart at the top of this README](#ta4j) for another example, or check the [wiki's charting guide](https://ta4j.github.io/ta4j-wiki/Charting.html) for more examples.
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)